### PR TITLE
S2U-14 5.1.1.10 Canceled questions and score re-calculation

### PR DESCRIPTION
**Pull request by stetsche**
Conversion scripts for:

- https://sakaiproject.atlassian.net/browse/S2U-14
- https://github.com/sakaiproject/sakai/pull/11328

### CODE REVIEW

**Comment by jesusmmp** [comment]
Title should be S2U-14.
@mpellicer Please add uni-digital label. I haven't got permission on this repo. Thanks!

**Comment by stetsche** [comment]
Thanks! Commit message was wrong too :)

File: docs/conversion/s2u_features/S2U-14-mysql.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- S2U-14 --
+ALTER TABLE SAM_PUBLISHEDITEM_T ADD CANCELLED INT(11) DEFAULT 0 NOT NULL;
```
Comment (mpellicer) [inline docs/conversion/s2u_features/S2U-14-mysql.sql:2]: Should we use a TINYINT for this using only 1 byte? We cover probably all the cases

File: docs/conversion/s2u_features/S2U-14-oracle.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- S2U-14 --
+ALTER TABLE SAM_PUBLISHEDITEM_T ADD CANCELLED NUMBER(10,0) DEFAULT 0 NOT NULL;
```
Comment (mpellicer) [inline docs/conversion/s2u_features/S2U-14-oracle.sql:2]: Seems too much for me to store something smaller than 100, do you know if number(2,0) is enough?

**Review by mpellicer — CHANGES_REQUESTED** [review]
Discussion around precission.

**Comment by mpellicer** [comment]
> Title should be S2U-14. @mpellicer Please add uni-digital label. I haven't got permission on this repo. Thanks!

Thanks!

File: docs/conversion/s2u_features/S2U-14-mysql.sql (hunk 1 above)
Comment (mpellicer) [reply docs/conversion/s2u_features/S2U-14-mysql.sql:2]: I think the name should be 'CANCELLATION', could you confirm?

**Review by mpellicer — APPROVED** [review]
Tested both scripts in MySQL and Oracle, good job so far!

**Pull request merged**